can_tools/scrapers/base.py

```diff
@@ -37,9 +37,17 @@ def __init__(
         self.sex = sex
 
 
+class RequestError(Exception):
```

Comment:
suggestion/thought for the future: as long as there are more than a few I've always liked having an `errors` module that makes it easy to reference common errors across different contexts.

Reply:
Yeah that works for me. so far I've just been collecting them here